src/OpenTelemetry.Extensions/README.md

```diff
@@ -84,3 +84,27 @@ var tracerProvider = Sdk.CreateTracerProviderBuilder()
 Warning: The baggage key predicate is executed for every baggage entry for each
 started activity.
 Do not use slow or intensive operations.
+
+### RateLimitingSampler
+
+The rate limiting sampler is a sampler that will limit the number of traces to
```

Comment:
is it counting the number of traces or number of spans (Activity)?

Reply:
It works on activities, but when used in conjunction with ParentBasedSampler, it becomes the number of traces. As its only called for the root spans, it does not cost or account for the number of spans in each trace.